partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py

```diff
@@ -207,44 +198,42 @@ def test_04_contact_creation(self):
         ctx = {'search_show_all_positions': {'is_set': True,
                                              'set_value': True
                                              }}
-        self.partner.unlink(cr, uid, [new_contact_id], context=ctx)
+        self.partner.unlink([new_contact_id])
         partner_ids = self.partner.search(
-            cr, uid, [('id', 'in', [new_contact_id, self.bob_contact_id])])
+            [('id', 'in', [new_contact_id, self.bob_contact_id])])
         self.assertIn(self.bob_contact_id, partner_ids)
         self.assertNotIn(new_contact_id, partner_ids)
 
     def test_05_contact_fields_sync(self):
         """Check that contact's fields are correctly synced between
         parent contact or related contacts
         """
-        cr, uid = self.cr, self.uid
 
         # Test DOWNSTREAM sync
         self.partner.write(
-            cr, uid, [self.bob_contact_id], {'name': 'Rob Egnops'}
+            [self.bob_contact_id], {'name': 'Rob Egnops'}
```

Comment:
Now that `self.bob_contact` is a recordset, you can directly use `self.bob_contact.name = 'Rob Egnops'`.